app/src/components/ChoiceCard.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ChoiceCard from './ChoiceCard'
+
+import '@testing-library/jest-dom';
+
+test('should render: title. Should not render: paragraph, image', () => {
```

Comment:
I meant something like:
```suggestion
it('should render: title. Should not render: paragraph, image', () => {
```

I didn't know that github has code suggestions, I only knew it on gitlab, thank you for show me that! :D